schema-derivatives-perpfutures.graphql

```diff
@@ -1797,6 +1797,15 @@ type PositionSnapshot @entity(immutable: true) {
   " Notional value of collateral position, in USD "
   collateralBalanceUSD: BigDecimal!
 
+  " Notional value of position right before position closed, in USD, only useful at close events. "
+  closeBalanceUSD: BigDecimal
+
+  " Notional value of collateral position ight before position closed, in USD, only useful at close events "
+  closeCollateralBalanceUSD: BigDecimal
+
+  " Realised profit and loss in USD at the time of close or event "
+  realisedPnlUSD: BigDecimal
+
```

Comment:
Doesn't it make more sense to add these to the `Position` and not the snapshot? This is only going to be recorded once, not necessary for each snapshot

Reply:
moved `closeBalanceUSD` and `closeCollateralBalanceUSD` to position not in snapshot. `realisedPnlUSD` i left in snapshot because it might be good to track during the lifetime of position.

Reply:
agreed, that could change throughout the life of the position. Would we also want to track it in the position as well?

Reply:
lets track in position as well. Added. Thanks for going through this and giving input. really valuable!